src/extension/src/extension.ts

```diff
@@ -5,19 +5,15 @@ import { PLATFORM } from "./constants/constants";
 
 export function activate(context: vscode.ExtensionContext): void {
   context.subscriptions.push(
-    vscode.commands.registerCommand(
-      "webTemplateStudioExtension.wizardLaunch.local",
-      async () => {
-        //TODO: Modify this to support React Native
-        Controller.getInstance(context, PLATFORM.WEB);
-      }
-    ),
-    vscode.commands.registerCommand(
-      "webTemplateStudioExtension.deployApp.local",
-      async () => {
-        await Deploy.getInstance(context).deployProject();
-      }
-    )
+    vscode.commands.registerCommand("webTemplateStudioExtension.web.createApp.local", async () => {
+      Controller.getInstance(context, PLATFORM.WEB);
+    }),
+    vscode.commands.registerCommand("webTemplateStudioExtension.reactNative.createApp.local", async () => {
+      Controller.getInstance(context, PLATFORM.REACTNATIVE);
+    }),
+    vscode.commands.registerCommand("webTemplateStudioExtension.deployApp.local", async () => {
```

Comment:
is it missing **web.** here? on the command?
```suggestion
    vscode.commands.registerCommand("webTemplateStudioExtension.web.deployApp.local", async () => {
```